plugins/generator-1.19.2/forge-1.19.2/mappings/stepsounds.yaml

```diff
@@ -0,0 +1,84 @@
+_default: GRAVEL
+GROUND: GRAVEL
+WOOD: WOOD
+PLANT: GRASS
+LILY_PADS: LILY_PAD
+STONE: STONE
+METAL: METAL
+GLASS: GLASS
+CLOTH: WOOL
+SAND: SAND
+SNOW: SNOW
+POWDER_SNOW: POWDER_SNOW
+LADDER: LADDER
+ANVIL: ANVIL
+SLIME: SLIME_BLOCK
+HONEY: HONEY_BLOCK
+WET_GRASS: WET_GRASS
+CORAL: CORAL_BLOCK
+BAMBOO: BAMBOO
+BAMBOO_SAPLING: BAMBOO_SAPLING
+SCAFFOLDING: SCAFFOLDING
+SWEET_BERRY_BUSH: SWEET_BERRY_BUSH
+CROP: CROP
+STEM: HARD_CROP
+VINE: VINE
+NETHER_WART: NETHER_WART
+LANTERN: LANTERN
+HYPHAE: STEM
+NYLIUM: NYLIUM
+FUNGUS: FUNGUS
+ROOT: ROOTS
+SHROOMLIGHT: SHROOMLIGHT
+NETHER_VINE: WEEPING_VINES
+NETHER_VINE_LOWER_PITCH: TWISTING_VINES
+SOUL_SAND: SOUL_SAND
+SOUL_SOIL: SOUL_SOIL
+BASALT: BASALT
+WART: WART_BLOCK
+NETHERRACK: NETHERRACK
+NETHER_BRICK: NETHER_BRICKS
+NETHER_SPROUT: NETHER_SPROUTS
+NETHER_ORE: NETHER_ORE
+BONE: BONE_BLOCK
+NETHERITE: NETHERITE_BLOCK
+ANCIENT_DEBRIS: ANCIENT_DEBRIS
+LODESTONE: LODESTONE
+CHAIN: CHAIN
+NETHER_GOLD: NETHER_GOLD_ORE
+GILDED_BLACKSTONE: GILDED_BLACKSTONE
+CANDLE: CANDLE
+AMETHYST_BLOCK: AMETHYST
+AMETHYST_CLUSTER: AMETHYST_CLUSTER
+SMALL_AMETHYST_BUD: SMALL_AMETHYST_BUD
+MEDIUM_AMETHYST_BUD: MEDIUM_AMETHYST_BUD
+LARGE_AMETHYST_BUD: LARGE_AMETHYST_BUD
+TUFF: TUFF
+CALCITE: CALCITE
+DRIPSTONE_BLOCK: DRIPSTONE_BLOCK
+POINTED_DRIPSTONE: POINTED_DRIPSTONE
+COPPER: COPPER
+CAVE_VINES: CAVE_VINES
+SPORE_BLOSSOM: SPORE_BLOSSOM
+AZALEA: AZALEA
+FLOWERING_AZALEA: FLOWERING_AZALEA
+MOSS_CARPET: MOSS_CARPET
+MOSS_BLOCK: MOSS
+BIG_DRIPLEAF: BIG_DRIPLEAF
+SMALL_DRIPLEAF: SMALL_DRIPLEAF
+ROOTED_DIRT: ROOTED_DIRT
+HANGING_ROOTS: HANGING_ROOTS
+AZALEA_LEAVES: AZALEA_LEAVES
+SCULK_SENSOR: SCULK_SENSOR
+GLOW_LICHEN: GLOW_LICHEN
+DEEPSLATE: DEEPSLATE
+DEEPSLATE_BRICKS: DEEPSLATE_BRICKS
+DEEPSLATE_TILES: DEEPSLATE_TILES
+POLISHED_DEEPSLATE: POLISHED_DEEPSLATE
+FROGLIGHT: FROGLIGHT
+FROGSPAWN: FROGSPAWN
+MANGROVE_ROOTS: MANGROVE_ROOTS
+MUDDY_MANGROVE_ROOTS: MUDDY_MANGROVE_ROOTS
+MUD: MUD
+MUD_BRICKS: MUD_BRICKS
+PACKED_MUD: PACKED_MUD
```

Comment:
`SCULK_CATALYST`, `SCULK`, `SCULK_VEIN` and `SCULK_SHRIEKER` are missing both in this mapping as well as in the data list.

Reply:
Good catch, fixed